client/src/js/services/ProjectService.js

```diff
@@ -23,13 +24,24 @@ function ProjectService($http, util) {
     .then(util.unwrapHttpResponse);
   }
 
+  function readComplete() {
+    var url = '/projects?complete=1';
+
+    return $http.get(url)
+    .then(util.unwrapHttpResponse);
+  }
+
   function update(id, project) {
     return $http.put('/projects/' + id, project)
     .then(util.unwrapHttpResponse);
   }
 
   function del(id) {
+
+    var result = confirm($translate.instant('PROJECT.CONFIRM'));
```

Comment:
You should use `$window.confirm` so that we can test it in the future.  [Relevant documentation](https://docs.angularjs.org/api/ng/service/$window).